cfg/types.go

```diff
@@ -31,11 +34,14 @@ func NewDefaultConfig() *Configuration {
 			Level: "info",
 		},
 		Symo: SymoConfig{
-			URL:              "http://symo.ip.or.hostname",
-			Timeout:          5 * time.Second,
-			Headers:          []string{},
-			PowerFlowEnabled: true,
-			ArchiveEnabled:   true,
+			URL:               "http://symo.ip.or.hostname",
+			Timeout:           5 * time.Second,
+			Headers:           []string{},
+			PowerFlowEnabled:  true,
+			ArchiveEnabled:    true,
+			SmartMeterEnabled: true,
```

Comment:
It may make sense to disable this endpoint by default. Otherwise, if there are devices that don't have this smart meter, an update to the next exporter version will have failing scrapes.
```suggestion
			SmartMeterEnabled: false,
```